modulegen/example_cmd.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Wdyt about creating a `cmd` package with the commands?

Reply:
Yes but where shall the generateModule be placed as it can’t stay in the main package in that case ?

Reply:
This require to move a lot a code. Let's keep it for another PR ?

Reply:
Mmm given this PR is adding cobra, I'd like to follow their project layout structure: https://github.com/spf13/cobra/blob/main/site/content/user_guide.md

```
  ▾ appName/
    ▾ cmd/
        add.go
        your.go
        commands.go
        here.go
      main.go
```

Reply:
I understand but this cannot be achieved without having first move out from main.go Context, Example, ExampleVar and every generate function . Otherwise you'll have a circular dependency.

Reply:
Let's do it in this PR,  the changes will be very narrowed to the module generator and with one single purpose: add cobra. I think it's fine if the PR gets a bit bigger